Add docs (#42)
ctg-lund/singleCellWorkflows: docs/10X-genomics/visium.md

# Introduction

> This is where I will document how to run spaceranger. It will start very manually and hopefully in the end mature into an automated pipeline. I will make no promises, but I plan to continously revise this guide as the workflow progress.

Reviewer: ```suggestion
This is a document describing how to run spaceranger. The document will be continuously updated while the pipeline matures.
```
